### ISSUE TITLE
What percentage of mass is in halos?

### ISSUE THREAD

**Issue by sweverett** [drphilmarshall/Pangloss#94]
Once #68 is finished, we should subtract our predicted `kappamap` (with and without smooth-component correction) with Stefan's to gauge how much of the mass in the Millennium Simulation we are missing by only using halos. We should also use the difference in maps to see if there is a better way to model the extra mass than with the smooth component correction.

**Comment by sweverett** [comment]
Also - we should compare our result with [this paper](http://arxiv.org/pdf/1503.04324v1.pdf) by Pace et al.

**Comment by drphilmarshall** [comment]
Right: we can provide illustrations to complement Pace et al's findings,
after we check them with the MS data we have.

**Comment by sweverett** [comment]
Here's the result of #96: (left is a binned Hilbert map, center is a binned lens-by-halos map, right is difference)
![kappamap_plots](https://cloud.githubusercontent.com/assets/8742179/15521877/f2a66150-21d3-11e6-88ff-a810ae8a4712.png)
While we are generally predicting individual features, we clearly are over-predicting the convergence overall. I thought this might be a result of the smooth-component correction, but the excess convergence between foreground mass is present in either case. This wasn't apparent in the early binned `kappamap` plots as the intensity scaling was changing - here all three plots have the same scaling.

**Comment by tcollett** [comment]
This figure is so cool and really useful! Great job.

It looks like our halo profiles aren't quite right. We seem to always over subtract in the centres of halos. What about playing around with the profiles to see if the differences can be shrunk with slightly different halo profiles? Maybe a first step is to keep the same profile but try decreasing the halo concentration.

**Comment by drphilmarshall** [comment]
Good suggestion from Tom here: re-titling and punting to the future.
